src/test/java/org/tikv/common/TiSessionTest.java

```diff
@@ -122,4 +125,29 @@ private void doCloseTest(boolean now, long timeoutMS) throws Exception {
       assertTrue(e.getMessage().contains("rejected from java.util.concurrent.ThreadPoolExecutor"));
     }
   }
+
+  @Test
+  public void warmUpTest() throws Exception {
+    TiConfiguration conf = createTiConfiguration();
+    conf.setWarmUpEnable(true);
+    doTest(conf);
+    conf.setWarmUpEnable(false);
+    doTest(conf);
+  }
+
+  private void doTest(TiConfiguration conf) throws Exception {
+    session = TiSession.create(conf);
+    long start = System.currentTimeMillis();
```

Comment:
is it better to use System.nanoTime()?

Reply:
I think it is not necessary in unit test.